### PR TITLE
change from hostPath mounts to TZ environment variable, update RBAC for finalizers

### LINKED ISSUE #35: remove hostPath mounts
Nice project, glad I came across it!

I've used helm install method, out of box the deployments will fail on OCP because hostPath mounts require elevated privileges. Perhaps the TZ could instead be set by environment variable and configured via helm values so that the user can configure if desired.

### PR DESCRIPTION
**Pull request by ctrought**
Fixes
- #35 
- #36 

I thought it might also be best to set image tag in helm values so they are fixed with each release.

### CODE REVIEW

**Comment by benjaminhuo** [comment]
@junotx would you help to review?

**Comment by benjaminhuo** [comment]
@junotx would you help to review?

File: Makefile [hunk 1]
```diff
@@ -36,6 +38,10 @@ generate: $(DEEPCOPY_TARGET) manifests
 manifests: $(CONTROLLER_GEN) $(TYPES_V1ALPHA1_TARGET)
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=operator paths="./pkg/..." output:crd:artifacts:config=config/crd/bases
 
+helm: $(CONTROLLER_GEN)
+	kustomize build config/helm | sed -e '/creationTimestamp/d' > helm/crds/bundle.yaml
+	tar zcvf kubernetes-event-exporter.tgz helm
```
Comment (junotx) [inline Makefile:43]: The tar package may be renamed `kube-events.tgz` as it includes not only the events exporter but also the events ruler.

**Review by junotx — APPROVED** [review]

**Review by benjaminhuo — APPROVED** [review]

**Comment by benjaminhuo** [comment]
@ctrought Thank you for this PR!

**Comment by benjaminhuo** [comment]
@junotx Maybe we can cut a new release now?

**Pull request merged**